tools/cdb2sql/cdb2sql.c

```diff
@@ -694,7 +695,9 @@ static void process_line(char *sql, int ntypes, int *types)
     sqlstr[len] = '\0';
 
     int start_time_ms, run_time_ms;
+    gbl_in_stmt = 1;
```

Comment:
What you are explaining is what I had implemented, until the consensus was to behave like bash and allow ctrl_c in a row to do nothing but just go to newline. Thus i removed that behavior.